end-to-end-tests-v2/package.json

```diff
@@ -9,6 +9,39 @@
     "createSmartCharacter:ci": "RPC_URL=$RPC_URL WORLD_ADDRESS=$WORLD_ADDRESS PRIVATE_KEY=$PRIVATE_KEY forge script script/CreateSmartCharacter.s.sol:CreateSmartCharacter --fork-url $RPC_URL --private-key $PRIVATE_KEY --broadcast --sig \"run(address)\" $WORLD_ADDRESS -vvv",
     "createSmartCharacter:local": "source ./.env.local && RPC_URL=$RPC_URL WORLD_ADDRESS=$WORLD_ADDRESS PRIVATE_KEY=$PRIVATE_KEY forge script script/CreateSmartCharacter.s.sol:CreateSmartCharacter --fork-url $RPC_URL --private-key $PRIVATE_KEY --broadcast --sig \"run(address)\" $WORLD_ADDRESS -vvv",
     "createSmartCharacter:devnet": "source ./.env.devnet && PRIVATE_KEY=$PRIVATE_KEY forge script script/CreateSmartCharacter.s.sol:CreateSmartCharacter --fork-url $RPC_URL --private-key $PRIVATE_KEY --broadcast --sig \"run(address)\" $WORLD_ADDRESS -vvv",
+    "mintERC20:local": "source ./.env.local && PRIVATE_KEY=$PRIVATE_KEY forge script script/MintERC20.s.sol:MintERC20 --fork-url $RPC_URL --private-key $PRIVATE_KEY --broadcast --sig \"run(address)\" $WORLD_ADDRESS -vvv",
+    "mintERC20:devnet": "source ./.env.devnet && PRIVATE_KEY=$PRIVATE_KEY forge script script/MintERC20.s.sol:MintERC20 --fork-url $RPC_URL --private-key $PRIVATE_KEY --broadcast --sig \"run(address)\" $WORLD_ADDRESS -vvv",
+    "createMintERC20:ci": "RPC_URL=$RPC_URL WORLD_ADDRESS=$WORLD_ADDRESS PRIVATE_KEY=$PRIVATE_KEY forge script script/CreateMintERC20.s.sol:CreateMintERC20 --fork-url $RPC_URL --private-key $PRIVATE_KEY --broadcast --sig \"run(address)\" $WORLD_ADDRESS -vvv",
```

Comment:
Does this require the addition of an `.env.ci` file?

Reply:
No ci does not need .env.ci. the ci job takes the default values